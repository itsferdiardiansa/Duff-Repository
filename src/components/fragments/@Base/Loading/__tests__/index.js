import { mount } from '@vue/test-utils';
import Loading from '@base/Loading';

describe('fragment/Base/Loading', () => {
  it('will render and match snapshot', async () => {
    const Component = mount(<Loading />);

    expect(Component.find('h1').html()).toContain('Please wait');
    expect(Component.html()).toMatchSnapshot();
  });
});
