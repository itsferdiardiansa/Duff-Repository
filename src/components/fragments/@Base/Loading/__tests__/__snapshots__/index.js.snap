// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fragment/Base/Loading will render and match snapshot 1`] = `
<div class="ss-loading fade-enter-from fade-enter-active">
  <div class="anim">
    <div></div>
    <div></div>
    <div></div>
  </div>
  <h1>Please wait</h1>
</div>
`;
